### PR TITLE
Athena/components

### PR DESCRIPTION
**Pull request by ath-chen**
- made responsive mobile pages 
- hid social buttons & photos tab & extra info section
- fixed stripe call with correct params

### CODE REVIEW

File: service-worker.js [hunk 1]
```diff
@@ -12,7 +12,11 @@
  */
 
 importScripts(
+<<<<<<< HEAD
+  "/precache-manifest.e74ba72f14a210479a5ccdeac672c65b.js"
+=======
   'https://storage.googleapis.com/workbox-cdn/releases/4.3.1/workbox-sw.js'
+>>>>>>> 11d092887a48b469799cd76017634269a1937815
```
Comment (stanleyzhengnyc) [inline service-worker.js:19]: this looks like a git conflict here

File: service-worker.js [hunk 2]
```diff
@@ -12,7 +12,11 @@
  */
 
 importScripts(
+<<<<<<< HEAD
```
Comment (jtmckibb) [inline service-worker.js]: @ath-chen looks like the merge conflict is still here

File: src/components/ModalPayment/ModalPayment.tsx [hunk 3]
```diff
@@ -30,9 +30,9 @@ const ModalPayment = ({
   const payment: PaymentParams = {
     amount: Number(donatedAmt) * 100,
     currency: 'usd',
-    name: 'Gift Card',
+    item_type: purchaseType === 'donation' ? 'donation' : 'gift_card',
```
Comment (jtmckibb) [inline src/components/ModalPayment/ModalPayment.tsx:33]: Would be nice to abstract these into an enum on a class in the future. Don't need to do now

File: src/components/ModalConfirmation/styles.module.scss [hunk 4]
```diff
@@ -21,3 +21,11 @@ form img {
...
   }
 }
+
+@media only screen and (max-width: 450px) {
+  .container {
+    width: 100%;
+    height: 100%;
+    overflow-x: hidden;
+  }
+}
```
Comment (jtmckibb) [inline src/components/ModalConfirmation/styles.module.scss:31]: You should run `yarn format` before you push to format everything. You should also run eslint too

File: service-worker.js (hunk 2 above)
Comment (ath-chen) [reply service-worker.js]: updated

File: service-worker.js (hunk 1 above)
Comment (ath-chen) [reply service-worker.js]: updated

File: src/components/ModalPayment/ModalPayment.tsx (hunk 3 above)
Comment (ath-chen) [reply src/components/ModalPayment/ModalPayment.tsx:33]: ya can update later on

**Pull request merged**